roles/clean-devstack-default-resources/tasks/main.yaml

```diff
@@ -5,22 +5,26 @@
       set -x
       source /opt/stack/new/devstack/openrc admin admin
       export OS_DOMAIN_NAME=Default
-      if [ "{{ global_env.OS_BRANCH }}" == "master" ]; then
-          # NOTE: workaround for issue #theopenlab/openlab/issues/101
-          pip install "openstacksdk==0.17.2"
+      export OS_DOMAIN_NAME=Default
+      if echo "stable/mitaka stable/newton" | grep -w "{{ global_env.OS_BRANCH }}" &>/dev/null; then
+          neutron router-gateway-clear router1
+          neutron router-interface-delete router1 ipv6-private-subnet
+          neutron router-interface-delete router1 private-subnet
+          neutron router-delete router1
+          openstack port delete `neutron port-list -f value -c id` || true
+      else
+          openstack router unset router1 --external-gateway
+          openstack router remove subnet router1 ipv6-private-subnet
+          openstack router remove subnet router1 private-subnet
+          openstack router delete router1
```

Comment:
why do not delete port at here, like: line 14?

Reply:
according to my testing, do not need, in testing with Mitaka, it need to delete ports

Reply:
would we can add a comment to make it clear? that will reduce confusion.

Reply:
And please add `Closes: theopenlab/openlab#101` into commit message

Reply:
ok, sure